unused_deps/unused_deps.go

```diff
@@ -116,9 +117,80 @@ func inputFileName(blazeBin, pkg, ruleName, extension string) string {
 	return fmt.Sprintf("%s/%s/%s.%s", blazeBin, pkg, ruleName, extension) // *_{binary,test}
 }
 
-// directDepParams returns --direct_dependency entries from paramsFileName (a jar-2.params file)
-// as a map from jar files to labels.
-func directDepParams(paramsFileNames ...string) (depsByJar map[string]string) {
+// manifestIndexNewline scans manifest and returns the index of the first
+// newline and the index of the byte following the newline.
+func manifestIndexNewline(manifest string) (int, int, error) {
+	// A newline in a jar manifest is denoted with CR LF, CR, or LF.
+	n := strings.IndexByte(manifest, '\n')
+	r := strings.IndexByte(manifest, '\r')
+	if n < 0 && r < 0 {
+		return -1, -1, fmt.Errorf("no newline in '%s'", manifest)
+	}
+
+	if n >= 0 {
```

Comment:
invert to reduce indent?

if n < 0 {
  return r, r + 1, nil
}
if n < r {
  ...